test/e2e/permissions.spec.js

```diff
@@ -163,6 +163,7 @@ describe('MetaMask', function () {
       await driver.switchToWindow(extension)
 
       await driver.clickElement(By.xpath(`//button[contains(text(), 'Disconnect All')]`))
+      await driver.clickElement(By.css('.popover-bg'))
```

Comment:
The confirmation modal is the old-style modal and is thus _behind_ the popover. This is a workaround pending the forthcoming redesign.